client/src/components/Listings/ProfileList/ProfileList.tsx

```diff
@@ -0,0 +1,27 @@
+import { Grid, Grow } from '@material-ui/core';
+
+import { useAuth } from '../../../context/useAuthContext';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+import ProfileCard from '../ProfileCard/ProfileCard';
+import LoadingSpinner from '../../LoadingSpinner/LoadingSpinner';
+
+export default function ProfileList(): JSX.Element {
+  const { updateSnackBarMessage } = useSnackBar();
+  const { userProfiles, loading, errorMsg } = useAuth();
+
+  if (loading) return <LoadingSpinner />;
+
+  if (errorMsg) updateSnackBarMessage(errorMsg);
+
+  return (
+    <Grow in={true}>
+      <Grid container spacing={8}>
+        {userProfiles.map((userProfile, index) => (
+          <Grid item xs={12} sm={4} key={index}>
```

Comment:
if we have userProfile, do we also have some other identifier like id that we can use as key?

Reply:
I must've failed to notice that I was using an index. I have changed to the profile's ID now. Thanks!